main.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+var Item = function(attribute) {
+  this.title = attribute.title;
+  this.complete = false;
+};
+
+var ToDoList = function() {
+  this.itemArr = [];
+};
+
+var myList = new ToDoList();
+
+
+ToDoList.prototype.addItem = function(item) {
+  this.itemArr.push(item);
+};
+
+Item.prototype.completeItem = function() {
+  this.complete = true;
+  $('ul').empty();
+  myList.renderToDom();
+};
+
+
+ToDoList.prototype.renderToDom = function() {
+  for (let i = 0; i < this.itemArr.length; i++) {
+    let currentItem = this.itemArr[i];
+    let li = $('<li/>');
+    if(this.itemArr[i].complete === true) {
```

Comment:
`this.itemArr[i].complete` will either be true or false, right? You don't need to check it equally to `true` then.